app/src/renderer/components/common/TmSessionAccountDelete.vue

```diff
@@ -120,7 +120,7 @@ export default {
   validations: () => ({
     fields: {
       deletionPassword: { required, minLength: minLength(10) },
-      deletionWarning: { required }
+      deletionWarning: { sameAs: sameAs(() => true) }
```

Comment:
I don't get why we need to implement a check that always return true

Reply:
And it seems on line 54 the tm-form-msg is checking agains the fields.deletionWarning.required check

Reply:
This is checking if the value is `true`:
https://github.com/monterail/vuelidate/issues/332#issuecomment-402993390